### DISCUSSION TITLE
How to make PR in forked project

### DISCUSSION THREAD

**Discussion by lunarforky-z** [community/community#143721] [Pull Requests]
### Select Topic Area

Question

### Body

i don't know how i can make PR in forked project

**Comment by hylcore-V** [comment]
after push ur commit,  u have to click the 'new pull request' button.
and then i have to select the purpose branch of the origin repository and ur current working branch in ur repository.
and submit it.

**Comment by lunarforky-z** [comment]
Thank you for your advice

**Comment by hylcore-V** [comment]
You are welcome
